.evergreen/config.yml

```diff
@@ -417,6 +417,7 @@ functions:
       type: test
       params:
         working_dir: "src"
+        silent: true
```

Comment:
Instead of silencing all test output, which makes it impossible to see why a run failed, could we silently load the SASL vars like we did in: https://github.com/mongodb/mongo-python-driver/pull/1072?

Reply:
Alternatively we can using include_expansions_in_env

Reply:
Updated to use the vault